concrete/controllers/search/groups.php

```diff
@@ -11,10 +11,11 @@
 
 class Groups extends Controller
 {
-    protected $fields = array();
+    protected $fields = [];
 
     public function __construct()
     {
+        parent::__construct();
```

Comment:
Needed to correctly initialize the instance (`$this->request` is not set otherwise)